example/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name">Android Adapters</string>
```

Comment:
I prefer `Realm adapter` since `Android` has no additional information on the device(apps installed in android device is all android application).